Solutions/MISP2Sentinel/SolutionMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-misp2sentinel",
+	"firstPublishDate": "2023-07-29",
+	"lastPublishDate": "2023-07-29",
+	"providers": ["MISP project", "cudeso.be"],
+	"categories": {
+		"domains" : ["Security - Threat Intelligence", "Security - Threat Protection"],
+		"verticals": ["Aeronautics", "Education", "Finance", "Healthcare", "Manufacturing", "Retail"]
+	},
+	"support": {
+	  "name": "MISP project & cudeso.be",
+	  "email": "info@cudeso.be",
+	  "tier": "Community-supported",

```

Comment:
The accepted tier values are Microsoft, Partner, Community. Please correct it. Thanks!

Reply:
changed in https://github.com/Azure/Azure-Sentinel/pull/8658/commits/01daea3dfc141fed17c97beae3d61d04e16749cb